lib/libzutil/os/freebsd/zutil_import_os.c

```diff
@@ -249,6 +249,13 @@ zfs_dev_flush(int fd)
 	return (0);
 }
 
+void
+update_vdev_config_dev_sysfs_path(nvlist_t *nv, const char *path, const char
+    *key)
+{
+	(void) path;
```

Comment:
You'll want to add a `(void) nv;` and `(void) key;` here as well to avoid the unused var